src/actions/ReceiveDropdown.tsx

```diff
@@ -71,7 +71,7 @@ export function showReceiveDropdown(navigation: NavigationBase, transaction: Edg
 
           navigation.navigate('transactionDetails', {
             edgeTransaction: transaction,
-            walletId
+            wallet
```

Comment:
We are not allowed to pass the wallet itself to a scene, but only the walletId. However, we have a functional middleware, `withWallet`, that will turn the `walletId` route param back into a `wallet` prop.

We do it this way so the app won't crash on auto-logout. The `withWallet` middlware handles all the crash cases.